### PR TITLE
Add services

### PR DESCRIPTION
**Pull request by JuliaFernee**
Basic configuration for a neo4j cluster with essential services such as splunk-forwarder, tunnel-registrator, diamond (for grafana), image-cleaner, neo4j-hot-backup and, of course, 3 instances of neo4j 3.1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,6 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+
+.idea/
```
Comment (nwrigh) [inline .gitignore:27]: *.idea ?

File: services.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  desiredState: loaded
+- name: neo4j-hot-backup@.timer
+  count: 1
+- name: splunk-forwarder.service
+- name: tunnel-registrator.service
+  version: v1.0.3
+- name: neo4j-sidekick@.service
+  count: 3
+- name: neo4j@.service
+  version: 3.1.0-rc1
```
Comment (nwrigh) [inline services.yaml]: rc, release candidate? We should have a stable version that is available now https://neo4j.com/download/other-releases/

**Review by nwrigh — COMMENTED** [review]
Also would mentioning the cluster in the name of this project be better? up-neo4j-cluster-service-files?

File: services.yaml (hunk 2 above)
Comment (nwrigh) [reply services.yaml]: Or are you waiting on the other task to put a docker image together?

File: neo4j@.service [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    --memory "10g" \
+    -v /vol/%p-%i/:/data -p 7474:7474 \
+    --env "NEO4J_AUTH=none" \
+    --env "NEO4J_HEAP_MEMORY=3584" \
+    --env "NEO4J_CACHE_MEMORY=1024m" \
+    --env "GRAPHITE_ENABLED=true" \
+    --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" \
+    --env "GRAPHITE_PREFIX=coco.services.$ENV.neo4j.%i" \
+    --env "NEO4J_causalClustering_expectedCoreClusterSize=3" \
+    --env "NEO4J_causalClustering_initialDiscoveryMembers=$NEO4J_causalClustering_initialDiscoveryMembers" \
```
Comment (nwrigh) [inline neo4j@.service]: Any reason why we are partially upper case and partially lowercase?

File: .gitignore (hunk 1 above)
Comment (ghost) [reply .gitignore:27]: IntelliJ config folder.

Comment (nwrigh) [reply .gitignore:27]: Isn't it *. though

**Pull request merged**